helix-core/src/surround.rs

```diff
@@ -0,0 +1,58 @@
+use crate::{search, Selection};
+use ropey::RopeSlice;
+
+pub const PAIRS: &[(char, char)] = &[('(', ')'), ('[', ']'), ('{', '}'), ('<', '>')];
```

Comment:
Let's add a few more which keyboard can usually type.
```suggestion
pub const PAIRS: &[(char, char)] = &[('(', ')'), ('[', ']'), ('{', '}'), ('<', '>'), ('«', '»'), ('「', '」), ('（', '）'), ('｛', '｝')];
```

Reply:
I've added them in. We should let this be extensible via the plugin system when it gets stabilized.